backend/routes/XrplNFTHelper.js

```diff
@@ -0,0 +1,462 @@
+import xrpl from 'xrpl';
+import axios from 'axios';
+import fs  from 'fs'; //used for storage of token metadata in devmode
+
+import XummSdk from 'xumm-sdk';
+
+
+
+const sdk = new XummSdk.XummSdk('d16951f8-6528-4c3f-88e9-37333e383421','e6d8a068-a55b-4e5a-b540-016847d7e376')

```

Comment:
Careful, I think this is a secret listed in a public repo - you'll want to use environment variables here instead. (Since GitHub stores history long-term, you'll want to get a new secret issued if you can, or go back and delete commits with this information in it)

You'll want to use a `.env` file like this: https://github.com/XRPL-Labs/XUMM-SDK/blob/master/.env.sample according to the docs on the npm page here: https://www.npmjs.com/package/xumm-sdk

Reply:
If you do use a `.env` file, make sure to include the steps for how to set it up clearly in the README (otherwise it can be really confusing to figure out what's broken in the configuration & how to fix it)